luxonis_train/attached_modules/losses/fomo_localization_loss.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from typing import Any
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+from luxonis_train.enums import TaskType
+from luxonis_train.nodes import FOMOHead
+from luxonis_train.utils import Labels, Packet
+
+from .base_loss import BaseLoss
+
+logger = logging.getLogger(__name__)
+
+
+class FOMOLocalizationLoss(BaseLoss[Tensor, Tensor]):
+    node: FOMOHead
+    supported_tasks: list[tuple[TaskType, ...]] = [
+        (TaskType.BOUNDINGBOX, TaskType.KEYPOINTS)
+    ]
+
+    def __init__(self, object_weight: float = 1000, **kwargs: Any):
```

Comment:
Original implementation has object_weight=100 as default. Does that not work as well?

Reply:
Feel free to set to the value that works best